com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -83,7 +83,7 @@ public enum LightRenderingMode
     public enum ShaderVariantLogLevel
     {
         Disabled,
-        OnlyUniversalRPShaders,
+        OnlyURPShaders,
```

Comment:
This is breaking change. Use API Upgrader.

Reply:
Thanks